src/contract-deployment/config.ts

```diff
@@ -215,11 +219,8 @@ export const makeContractDeployConfig = async (
     OVM_ETH: {
       factory: getContractFactory('OVM_ETH'),
       params: [
-        AddressManager.address,
-        config.ethConfig.initialAmount,
-        'Ether',
-        18,
-        'ETH',
+        '0x4200000000000000000000000000000000000007',
+        '0x0000000000000000000000000000000000000000', // will be overridden by geth when state dump is ingested.  Storage key: 0x0000000000000000000000000000000000000000000000000000000000000008
```

Comment:
We should definitely make a single source of truth for all of these pre-deploy addresses. If we are already hardcoding the addresses then I don't think we need to change it for this PR, but we should definitely at least add an issue for cleaning this up